yarn-project/circuit-types/src/interfaces/pxe.ts

```diff
@@ -302,6 +303,12 @@ export interface PXE {
    */
   getSyncStatus(): Promise<SyncStatus>;
 
+  /**
+   * Returns the note processor stats.
+   * @returns The note processor stats for notes for each public key being tracked.
+   */
+  getSyncStats(): Promise<{ [key: string]: NoteProcessorStats }>;
+
```

Comment:
Added this getter as I found no other way to extract whether the note processor decrypted a log then dropped it. I wanted to be more exact than simply 'no notes were added' but can revert this if it doesn't seem that useful!